cmd/do-csi-plugin/main.go

```diff
@@ -40,6 +40,7 @@ func main() {
 		defaultVolumesPageSize = flag.Uint("default-volumes-page-size", 0, "The default page size used when paging through volumes results (default: do not specify and let the DO API choose)")
 		doAPIRateLimitQPS      = flag.Float64("do-api-rate-limit", 0, "Impose QPS rate limit on DigitalOcean API usage (default: do not rate limit)")
 		version                = flag.Bool("version", false, "Print the version and exit.")
```

Comment:
Let's keep the `version` flag last in the list since it's usually the least important.